index.js

```diff
@@ -1,135 +1,35 @@
 #! /usr/bin/env node
-const path  = require('path');
-const fs    = require('fs');
 const yargs = require('yargs');
-const chalk = require('chalk');
 
-const Select    = require('./lib/test/select');
-const Operators = require('./lib/infra/operators');
-const Steps     = require('./lib/read/stepsReader');
+const Stepper = require('./lib/read/stepper');
+const Reporter = require('./lib/read/reporter');
 
-(async()=>{
+(async () => {
 
-    yargs.command('test <stepfile>', 'Test markdown file', (yargs) => { }, async (argv) => {
-
-        await testreport("test", argv);
-
-    });
-
-    yargs.command('report <stepfile>', 'Test markdown file and report feedback into rendered output', (yargs) => { }, async (argv) => {
-
-        await testreport("report", argv);
-
-    });
-
-    yargs.command('notebook <notebook>', 'Test notebook html and report feedback into rendered output', (yargs) => { }, async (argv) => {
-        await notebook(argv);
+    yargs.command('report <doc> [html]', 'Test markdown/steps file and report feedback into rendered output', (yargs) => { }, async (argv) => {
+        await docable(argv, true);
     });
 
     // Turn on help and access argv
     yargs.help().argv;
 
 })();
 
-async function notebook(argv) {
-    const notebookPath = argv.notebook.startsWith('/') ? argv.notebook : path.join(process.cwd(), argv.notebook);
-    let notebook;
-    try {
-        notebook = await fs.promises.readFile(notebookPath)
-    }
-    catch (err) {
-        console.error('Error:', err);
-        process.exit(1);
-    }
-
-    let stepper = new Steps();
-    let { conn, cells, $ } = await stepper.readNotebook(notebook);
-    const cwd = process.cwd();
-
-    const op = new Operators(conn, '.');
-
-    for (const cell of cells) {
-        let result;
-        switch (cell.type) {
-            case 'file':
-                result = await op.file(cell.content, cell.path, cell.user);
-                break;
-            case 'command':
-                result = await op.run(cell.content, cell.user, cell.persistent);
-                break;
-            default:
-                break;
-        }
-
-        console.log('results:', result);
+async function docable(argv, report) {
+    let stepper = new Stepper(argv.doc, argv.html);
+    await stepper.setup(argv.doc, argv.html);
```

Comment:
We shouldn't have to pass `(doc, html)` in constructor _and_ `.setup(doc,html)`. 
Pick one.

Reply:
oops, `async setup() {` already doesn't get any arguments. Forgot to remove them from `.setup()` call when I was changing things.

Reply:
fixed in 339f5c0.